packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -122,7 +122,7 @@ Future<XcodeBuildResult> buildXcodeProject({
     return XcodeBuildResult(success: false);
   }
 
-  await removeFinderExtendedAttributes(app.project.hostAppRoot, globals.processUtils, globals.logger);
+  await removeFinderExtendedAttributes(app.project.parent.directory, globals.processUtils, globals.logger);
```

Comment:
Instead of removing from the `IosProject` directory, remove from the entire project directory.